docs/csharp/tutorials/nullable-reference-types.md

```diff
@@ -146,15 +132,15 @@ Switch to `Program.cs` in your editor and replace the contents of `Main` with th
 
 [!code-csharp[AddQuestions](../../../samples/csharp/NullableIntroduction/NullableIntroduction/Program.cs#AddQuestions)]
 
-Without the `#nullable enable` pragma at the top of the file, the compiler doesn't issue a warning when you pass `null` as the text for the `AddQuestion` argument. Fix that by adding `#nullable enable` following the `using` statement. Try it by adding the following line to `Main`:
+Because the entire project is a nullable enabled context, you'll get warnings when you pass `null` to any method expecting a non-nullable reference type. Try it by adding the following line to `Main`:
```

Comment:
"the entire project is a nullable enabled context" => "the entire project is in a nullable enabled context"? Or "the nullable context is enabled for the entire project"?
(It's possible that what you've written is exactly what you intended - contexts can be like that, I guess. It just reads a little oddly to me.)